.expeditor/verify.pipeline.yml

```diff
@@ -18,17 +18,177 @@ steps:
 
   - label: "[lint] :linux: :bash: rustfmt"
     command:
+      - ./test/run_cargo_test.sh builder-jobsrv # this is overkill, we just need the build
```

Comment:
Do we need to build all of jobsrv, or can we just build builder-protocol?

Reply:
Unfortunately the whole thing. I tried `builder-protocol` first and it didn't build out the things in the correct locations.